src/model/cloud-runner/cloud-runner.test.ts

```diff
@@ -0,0 +1,46 @@
+import { BuildParameters, ImageTag } from '..';
+import CloudRunner from './cloud-runner';
+import Input from '../input';
+import fs from 'fs';
+import { CloudRunnerState } from './state/cloud-runner-state';
+import { CloudRunnerStatics } from './cloud-runner-statics';
+
+describe('Cloud Runner', () => {
+  it('responds', () => {});
+});
```

Comment:
Probably can be removed

Reply:
Jest requires a tests file to always complete at least one test. When the cloud runner tests are disabled the main test below doesn't run. 

Jest then complains no tests complete in this file... :(